exodus/exodus/settings/common_dev.py

```diff
@@ -3,31 +3,29 @@
 from .base import *
 
 default_secret_key = '9b80473f1b0c7d9f1859cfa754e40e26'
-SECRET_KEY = os.environ.get('EXODUS_SECRET_KEY', default_secret_key)
+SECRET_KEY = env('EXODUS_SECRET_KEY', default=default_secret_key)
 
-DEBUG = os.environ.get('DEBUG', 'True').lower() == 'true'
 ALLOWED_HOSTS = ['*']
 
 STATIC_URL = '/static/'
-STATIC_ROOT = '/static/'
-STATICFILES_DIRS = (os.path.join(BASE_DIR, '..', 'static'), )
+STATIC_ROOT = f'{BASE_DIR}/static/'
+STATICFILES_DIRS = [str(APPS_DIR / "static")]
 
-MINIO_STORAGE_ACCESS_KEY = os.environ.get('EXODUS_MINIO_ROOT_USER', 'exodusexodus')
-MINIO_STORAGE_SECRET_KEY = os.environ.get('EXODUS_MINIO_ROOT_PASSWORD', 'exodusexodus')
+MINIO_STORAGE_ACCESS_KEY = env('EXODUS_MINIO_ROOT_USER', default='exodusexodus')
+MINIO_STORAGE_SECRET_KEY = env('EXODUS_MINIO_ROOT_PASSWORD', default='exodusexodus')
 
 ALLOW_APK_UPLOAD = True
 
 TRACKERS_AUTO_UPDATE = True
 
-customization = os.environ.get('EXODUS_CUSTOMIZATION')
+customization = env('EXODUS_CUSTOMIZATION', default='')
 if customization:
     INSTALLED_APPS = [customization] + INSTALLED_APPS
 
-google_username = os.environ.get('EXODUS_GOOGLE_USERNAME')
-google_password = os.environ.get('EXODUS_GOOGLE_PASSWORD')
+google_username = env('EXODUS_GOOGLE_USERNAME', default='')
+google_password = env('EXODUS_GOOGLE_PASSWORD', default='')
 if google_username and google_password:
     GOOGLE_ACCOUNT_USERNAME = google_username
     GOOGLE_ACCOUNT_PASSWORD = google_password
 
-if os.environ.get('EXODUS_CSRF_COOKIE_SECURE') == "False":
-    CSRF_COOKIE_SECURE = False
+CSRF_COOKIE_SECURE = env.bool('EXODUS_CSRF_COOKIE_SECURE', default=False)
```

Comment:
I think the default should be True

```suggestion
CSRF_COOKIE_SECURE = env.bool('EXODUS_CSRF_COOKIE_SECURE', default=True)
```

Reply:
Okay

Reply:
Done